.github/workflows/CI.yml

```diff
@@ -42,31 +42,39 @@ jobs:
 
   make_mason:
     runs-on: ubuntu-latest
-    env:
-      CHPL_SMOKE_SKIP_MAKE_MASON: false
     steps:
     - uses: actions/checkout@v2
     - name: make mason
+      # Use a quickstart config to keep it from running to long
+      # While there, run a make check in that config for more coverage
       run: |
-        CHPL_LLVM=none ./util/buildRelease/smokeTest
+        ./util/buildRelease/smokeTest quickstart mason chpl
```

Comment:
The `chpl` target running `make check` probably only takes ~10s so it probably doesn't matter but in the current checks it looks like `check_docs` is a little faster if we wanted to put that check there.

Reply:
Seems like a wash at the present time. (In the last build, make_doc took 6m47s and make_mason took 6m52s -- so moving a 10s job from make_mason to make_docs would just make them swap positions but not really change the overall situation).